commands/1_clicks.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2018 The Doctl Authors All rights reserved.
```

Comment:
I know this is from a copy-paste, but could you switch the copyright year here to 2020?